manifests/p/Postman/Postman/10.9.3/Postman.Postman.installer.yaml

```diff
@@ -0,0 +1,29 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Postman.Postman
+PackageVersion: 10.9.3
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: exe
+  Scope: user
+  InstallerUrl: https://dl.pstmn.io/download/version/10.9.3/win64
+  InstallerSha256: 9a85c5959449365ad2800bfc5dfa7b8ba0d350bdd7e1602bbab17f1ab01ecda7
+  InstallerSwitches:
+    Silent: -s
+    SilentWithProgress: -s
+  UpgradeBehavior: install
+  ProductCode: Postman
+  AppsAndFeaturesEntries:
+  - DisplayName: Postman x86_64 10.9.3
+    DisplayVersion: 10.9.3
+    ProductCode: Postman
```

Comment:
```suggestion
```
ARP entries especially with DisplayVersion are not needed here as PackageVersion is already equal to DisplayVersion and upgrade matching should work without them as well.

@rodrigoborgesdeoliveira I believe you added the entries in the last two manifests. We shouldn't be using DisplayVersion here as it will cause unnecessary pre-checks (See https://github.com/microsoft/winget-pkgs/pull/65816#issue-1301301147) and here it will also lead to infinite upgrade scenarios for the user if previous manifests do not use this field (See https://github.com/microsoft/winget-pkgs/issues/88726#issuecomment-1398993435).
We should remove them from the previous two manifests as well as they would likely cause the above mentioned infinite upgrades for user.

Reply:
Got it. Thanks for the info @mdanish-kh! I'll update the previous manifests later today and post a comment here once those are ready.

Reply:
To confirm, is it recommended to not have the `AppsAndFeaturesEntries` in these situations at all or just not have the `DisplayVersion`?

Reply:
I've opened the PRs for versions 10.8.0 and 10.9.0:
- #95932
- #95934

@mdanish-kh, the URL for version 10.9.0 doesn't work for me anymore even through `winget install`. Please let me know if you can still download it after a few hours have passed. I'm wondering if it has anything to do with CDNs or something similar.